fix: extract shared logic for server island loading into a function and only include it once (#13576)
withastro/astro: .changeset/bumpy-onions-mix.md

---
'astro': patch
---

> Extract shared logic for server island loading into a function and only include it once. This reduces duplicate code in the HTML.

Author: ```suggestion
Reduces duplicate code in server islands scripts by extracting shared logic into a helper function.
```
